projects/packages/calypsoify/src/class-jetpack-calypsoify.php

```diff
@@ -0,0 +1,215 @@
+<?php
+/**
+ * This is Calypso skin of the wp-admin interface that is conditionally triggered via the ?calypsoify=1 param.
+ *
+ * @package automattic/jetpack-calypsoify
+ */
+
+namespace Automattic\Jetpack\Calypsoify;
+
+use Automattic\Jetpack\Status;
+
+/**
+ * Class Jetpack_Calypsoify
+ */
+class Jetpack_Calypsoify {
+
+	const PACKAGE_VERSION = '0.1.0-alpha';
+
+	/**
+	 * Singleton instance of `Jetpack_Calypsoify`.
+	 *
+	 * @var object
+	 */
+	public static $instance = false;
+
+	/**
+	 * Is Calypsoify enabled, based on any value of `calypsoify` user meta.
+	 *
+	 * @var bool
+	 */
+	public $is_calypsoify_enabled = false;
+
+	/**
+	 * Jetpack_Calypsoify constructor.
+	 */
+	private function __construct() {
+		add_action( 'admin_init', array( $this, 'setup' ), 4 );
+	}
+
+	/**
+	 * Singleton.
+	 *
+	 * @return Jetpack_Calypsoify
+	 */
+	public static function get_instance() {
+		if ( ! self::$instance ) {
+			self::$instance = new self();
+		}
+
+		return self::$instance;
+	}
+
+	/**
+	 * Setup function that is loaded on the `wp_loaded` hook via the constructor.
+	 */
+	public function setup() {
+		$this->is_calypsoify_enabled = isset( $_GET['calypsoify'] ) && 1 === (int) $_GET['calypsoify'] && $this->is_page_gutenberg(); // phpcs:ignore WordPress.Security.NonceVerification.Recommended
+
+		$this->check_meta();
+
+		if ( $this->is_calypsoify_enabled ) {
+			add_action( 'admin_enqueue_scripts', array( $this, 'enqueue_for_gutenberg' ), 100 );
+		}
+	}
+
+	/**
+	 * Enqueues scripts, data, and styles for Gutenberg.
+	 */
+	public function enqueue_for_gutenberg() {
+		$site_suffix = ( new Status() )->get_site_suffix();
+		wp_enqueue_style( 'calypsoify_wpadminmods_css', plugin_dir_url( __FILE__ ) . 'build/style-gutenberg.css', false, self::PACKAGE_VERSION );
+		wp_style_add_data( 'calypsoify_wpadminmods_css', 'rtl', 'replace' );
+		wp_style_add_data( 'calypsoify_wpadminmods_css', 'suffix', '.min' );
+
+		wp_enqueue_script( 'calypsoify_wpadminmods_js', plugin_dir_url( __FILE__ ) . 'build/mods-gutenberg.js', array( 'jquery' ), self::PACKAGE_VERSION, false );
```

Comment:
After changing the paths, I will need to double check this is all working correctly once hooking everything up in the next PR, but this should be ok to leave as is for this PR.
For example the .min file is actually the current .css file, so I may need to remove this line or change the build output but will investigate more later.

Reply:
Yeah I'm also not sure what is the best way to handle `rtl`.
Based on my understanding so far, webpack by default will create an rtl file for eg `style.css` named `style.rtl.css` however [wordpress](https://codex.wordpress.org/Right-to-Left_Language_Support) expects `style-rtl.css` when setting the `rtl` property.

Reply:
Ideally the JS files should be built with our normal monorepo webpack config and then we should use our jetpack-assets package to enqueue things with `Assets::register_script()`. That handles a lot of the various bits for us.

I have no idea whether making that happen here would be a lot of work though.